### ISSUE TITLE
A "fixed" Table of contents `aside` wastes space

### ISSUE THREAD

**Issue by adelton** [validatedpatterns/docs#334]
The pattern pages like https://validatedpatterns.io/patterns/industrial-edge/ seem to contain
```
<aside class="pf-c-jump-links pf-m-vertical sticky pf-m-expandable pf-m-non-expandable-on-2xl" aria-label="Table of contents"> ...
```
on the right which is empty and wastes space.

![Screenshot_2023-10-30_13-09-01](https://github.com/validatedpatterns/docs/assets/2536912/afac5cf6-d23a-4fae-ae0e-59ca3329d6a7)

Moreover, when one tries to increase the size of the text with `Ctrl-+`, that white space grows and wastes even more space.

![Screenshot_2023-10-30_13-09-22](https://github.com/validatedpatterns/docs/assets/2536912/ddd2e0ac-5b3f-4529-9d84-9755481dbbe7)

It should be turned to `float` or removed completely.

**Comment by adelton** [comment]
Note to self -- the following Violentmonkey script workarounds the problem:

```
// ==UserScript==
// @name        Workaround https://github.com/validatedpatterns/docs/issues/334
// @namespace   adelton
// @match       https://validatedpatterns.io/patterns/*
// @grant       none
// @version     1.0

// ==/UserScript==

var TOC = document.getElementById('TableOfContents');
if (TOC) {
  TOC.parentNode.style.display = 'none';
}
```

**Comment by mbaldessari** [comment]
@abhatt-rh can you take a look at this?

**Comment by danmacpherson** [comment]
Hi @adelton

So that column should not be empty and should have table of contents for the page. For example:

![image](https://github.com/validatedpatterns/docs/assets/1260665/124c268b-a1f0-4649-b662-1aac2603acb2)

I might need some help to replicate this issue so I can work out the right way to fix it. Can you let me know the following:

- Your browser and browser version
- Any browser extensions installed
- Your screen resolution
- Your operating system

**Comment by adelton** [comment]
- firefox-119.0-2.fc39.x86_64
- multiple but you can assume none because I can see the same issue in a private window as well where extensions are disabled
- 1920 x 1080
- Fedora 39

**Comment by danmacpherson** [comment]
Thanks, @adelton. The good news is I'm running the exact same configuration you are, so it might be easier to isolate the issue.

I have a feeling the page zoom is affecting the TOC column, so I might need to correct this. Just to confirm, do you usually have the pages in Firefox zoomed in? If you click View > Zoom > Actual Size or Zoom Out, does the menu reappear?

**Comment by adelton** [comment]
Yes, after zooming out to a point when the text is so small I cannot really read it, the menu appears.
![Screenshot_2023-11-13_17-04-21](https://github.com/validatedpatterns/docs/assets/2536912/1d89bc4b-df4f-4159-af2e-f9ee03ffe936)

**Comment by danmacpherson** [comment]
Excellent! Seems like we've got the root cause. I'll try and implement a fix to make that right column respect the zoom.
